Assets/Plugins/Source/Editor/Build/BuildRunner.cs

```diff
@@ -125,5 +130,55 @@ public void OnPostprocessBuild(BuildReport report)
             // Perform the post-build task for the platform using it's builder.
             GetBuilder()?.PostBuild(report);
         }
+
+        /// <summary>
+        /// Checks to make sure that the platform configuration file exists where it is expected to be
+        /// TODO: Add configuration validation.
+        /// </summary>
+        private static void CheckPlatformConfiguration()
+        {
+            string configFilePath = PlatformManager.GetConfigFilePath();
+            if (!File.Exists(configFilePath))
+            {
+                throw new BuildFailedException($"Expected config file \"{configFilePath}\" for platform {PlatformManager.GetFullName(PlatformManager.CurrentPlatform)} does not exist.");
+            }
+        }
+
+        /// <summary>
+        /// Completes all configuration tasks.
+        /// </summary>
+        private static void ConfigureVersions()
+        {
+            // TODO: DANGEROUS WAITING. There Isn't a reason I can think of that AutoSetProductVersion
+            //       task would fail, but that's just the kind of thing that could end up in an inf 
+            //       loop
+            AutoSetProductVersion().Wait(10000);
```

Comment:
Fix this so that either all the calls to `Wait` have a default timeout, or this timeout is removed. "Waiting" is only really dangerous if you _don't_ have a timeout. As always, need to guarantee that exceptions are handled with grace.